frontend/src/components/DesktopApps/Screens/SettingsAppScreen.tsx

```diff
@@ -0,0 +1,13 @@
+import { resetGameData } from "../../../slices/gameDataSlice";
+import { useDispatch } from "react-redux";
+import { resetUpgrades } from "../../../slices/upgradesSlice";
+
+export default function SettingsAppScreen() {
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      <button onClick={() => confirm("test") && dispatch(resetGameData()) && dispatch(resetUpgrades())}>Reset</button>
```

Comment:
This confirm message should be changed to something more useful, not just "test".

Reply:
Changed to an actual confirm message.